__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => db.end());
+
+describe("*", () => {
+  test("GET 404: responds with error message if API is not found", () => {
+    return request(app)
+      .get("/notARoute")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Not Found");
+      });
+  });
+});
+
+describe("/api/topics", () => {
+  test("GET 200 responds with array of topic objects with properties 'slug' and 'description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics.length).toBe(3);
+        response.body.topics.forEach((topic) => {
+          expect(typeof topic.slug).toBe("string");
+          expect(typeof topic.description).toBe("string");
+        });
+      });
+  });
```

Comment:
Very nice test, double check whether there is also an `id` property returned ( I can't quite remember if there are more properties that come with this query, however if there are they should ideally all be tested )